src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -838,18 +838,10 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					Command = commandsViewModel.CreateFolderWithSelection,
 					ShowItem = itemsSelected,
 				},
-				new ContextMenuFlyoutItemViewModel()
+				new ContextMenuFlyoutItemViewModelBuilder(commands.CreateShortcut)
 				{
-					Text = "BaseLayoutItemContextFlyoutShortcut/Text".GetLocalizedResource(),
-					OpacityIcon = new OpacityIconModel()
-					{
-						OpacityIconStyle = "ColorIconShortcut",
-					},
-					Command = commandsViewModel.CreateShortcutCommand,
-					ShowItem = itemsSelected && (!selectedItems.FirstOrDefault()?.IsShortcut ?? false),
-					SingleItemOnly = true,
```

Comment:
Just noticed this was removed, create shortcut should only show if a single item is selected.